docs/contents/components/forms/month-picker/index.mdx

```diff
@@ -28,59 +28,91 @@ import { MonthPicker } from "@yamada-ui/calendar"
 
 ```tsx
 <VStack>
-  <MonthPicker variant="outline" placeholder="outline" />
-  <MonthPicker variant="filled" placeholder="filled" />
-  <MonthPicker variant="flushed" placeholder="flushed" />
-  <MonthPicker variant="unstyled" placeholder="unstyled" />
+  <For each={["outline", "filled", "flushed", "unstyled"]}>
+    {(variant, index) => (
+      <MonthPicker key={index} variant={variant} placeholder={variant} />
+    )}
+  </For>
 </VStack>
 ```
 
 To change the variant of the calendar, set the `variant`.
 
 ```tsx
 <VStack>
-  <MonthPicker placeholder="solid" calendarVariant="solid" />
-  <MonthPicker placeholder="subtle" calendarVariant="subtle" />
+  <For each={["solid", "subtle"]}>
+    {(calendarVariant, index) => (
+      <MonthPicker
+        key={index}
+        placeholder={calendarVariant}
+        calendarVariant={calendarVariant}
+      />
+    )}
+  </For>
 </VStack>
 ```
 
 ### Change Size
 
 ```tsx
 <VStack>
-  <MonthPicker placeholder="extra small size" size="xs" />
-  <MonthPicker placeholder="small size" size="sm" />
-  <MonthPicker placeholder="medium size" size="md" />
-  <MonthPicker placeholder="large size" size="lg" />
+  <For
+    each={[
+      { placeholder: "extra small size", size: "xs" },
+      { placeholder: "small size", size: "sm" },
+      { placeholder: "medium size", size: "md" },
+      { placeholder: "large size", size: "lg" },
+    ]}
+  >
+    {({ placeholder, size }, index) => (
+      <MonthPicker key={index} placeholder={placeholder} size={size} />
+    )}
+  </For>
 </VStack>
 ```
 
 To change the size of the calendar, set the `calendarSize`.
 
 ```tsx
 <VStack>
-  <MonthPicker placeholder="small size" calendarSize="sm" />
-  <MonthPicker placeholder="medium size" calendarSize="md" />
-  <MonthPicker placeholder="large size" calendarSize="lg" />
+  <For
+    each={[
+      { placeholder: "small size", calendarSize: "sm" },
+      { placeholder: "medium size", calendarSize: "md" },
+      { placeholder: "large size", calendarSize: "lg" },
+    ]}
+  >
+    {({ placeholder, calendarSize }, index) => (
+      <MonthPicker
+        key={index}
+        placeholder={placeholder}
+        calendarSize={calendarSize}
+      />
+    )}
+  </For>
```

Comment:
During this week's regular meeting, Mr. Yamada mentioned, "Please avoid using the `For` component if it reduces readability compared to the existing code or makes the code longer." I believe this is a case where the `For` component should not be used. @nathanlao

Reply:
@kaehehehe @illionillion 
Thank you for letting me know! I have re-reviewed the code and reverted most of the changes that make the code significantly longer, and only leave the variant part. Please let me know if this is not what we expected. If so I will close the PR.

Reply:
Thank you for your good work. :)